packages/material-ui/src/Typography/Typography.js

```diff
@@ -164,6 +164,27 @@ function getVariant(theme, props, variant) {
   return variant;
 }
 
+const defaultHeadlingMapping = {
+  h1: 'h1',
```

Comment:
typo:
```diff
- defaultHeadlingMapping
+ defaultHeadlineMapping
```

Reply:
Thank.

*P.S. it would be better to comment on the right line so GitHub can hide it once it's fixed.*

Reply:
I keep missing that for some reason.